src/jormungandr/testing/jormungandr-automation/src/jormungandr/starter/params/builder.rs

```diff
@@ -45,6 +45,21 @@ impl Default for JormungandrBootstrapper {
 }
 
 impl JormungandrBootstrapper {
+    pub fn default_with_config(node_config: NodeConfig) -> Self {
```

Comment:
I would like to have a documentatio part why it exist. Basically what you wrote on PR is great so IMHO we should just pass this explanation here